pkg/interceptors/github/github.go

```diff
@@ -96,7 +126,270 @@ func (w *Interceptor) Process(ctx context.Context, r *triggersv1.InterceptorRequ
 		}
 	}
 
+	// For event types pull_request, issue_comment check github owners approval
```

Comment:
```suggestion
	// For event types pull_request, issue_comment check github owners approval is required
	// User can specify both event type or any one of them
```
Let me know if that is not clear